os/drivers/bluetooth/realtek_ble/Kconfig

```diff
@@ -0,0 +1,34 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config BLE_CENTRAL_CONN_NUM
+	int "BLE Central Connection"
+	default 3
+	---help---
+		Set the Maximum number of BLE Central Connection
+
+config BLE_PERIPHERAL_CONN_NUM
+	int "BLE Peripheral Connection"
+	default 1
+	---help---
+		Set the Maximum number of BLE Peripheral Connection
+
+config BLE_MAX_BONDED_DEVICE
+	int "BLE Max Bonded Device"
+	default 5
+	---help---
+		Set the Maximum number of BLE Bond Data
```

Comment:
Does BLE Bond data mean BLE Bonded device only ?

Reply:
Yes. That's right.
BLE driver only stores a mac address of bonded device.